nearcore/src/dyn_config.rs

```diff
@@ -45,7 +46,14 @@ pub fn read_updateable_configs(
 pub fn get_updateable_client_config(config: Config) -> UpdateableClientConfig {
     // All fields that can be updated while the node is running should be explicitly set here.
     // Keep this list in-sync with `core/dyn-configs/README.md`.
-    UpdateableClientConfig { expected_shutdown: config.expected_shutdown }
+    UpdateableClientConfig {
+        expected_shutdown: config.expected_shutdown,
+        validator_signer: if config.update_key_file.unwrap_or_default() {
+            InMemoryValidatorSigner::from_file(&Path::new(&config.validator_key_file)).ok()
```

Comment:
This is looking for the validator key file in the wrong directory.
Should be `home_dir.join(config.validator_key_file)`.

Reply:
Agree, the absolute path should be used here.
Thanks for pointing that out.